src/patterns/error-validation/examples/errors-proto/errors/index.njk

```diff
@@ -281,12 +276,13 @@ layout: none
             onsInput({
                 "id": "email",
                 "type": "email",
+                "value": "name.com",
```

Comment:
Would this be better with an example email address instead of just a domain?

Reply:
The point is to imitate an invalid email address error